### PR TITLE
#164424155 Allow all users (authenticated or not) to access tags 

### PR DESCRIPTION
**Pull request by kdaglas**
#### What does this PR do?
This PR enables any user logged in or not to access tags so as to easily filter 
fix bug preventing users from liking more than one comment or reply

#### Description of Task to be completed?
- should have the endpoint `/api/tags` working
- remove authentication
- add a check for if a user already like a particular comment or reply 

#### How should this be manually tested?
- Fetch this branch by running the command `git fetch origin bg-change-tag-auth-164424155` in the terminal and check into the branch `git checkout bg-change-tag-auth-164424155`
- Start the virtual environment `source venv/bin/activate`
- Migrate so that the fields required for the data are initiated by typing this command: `python manage.py migrate`
- Start the application by running this command: `python manage.py runserver`
- Go to postman and fetch all the tags

#### Screenshots
[#164424155](https://www.pivotaltracker.com/story/show/164424155)

### CODE REVIEW

**Review by calvinpete — DISMISSED** [review]
LGTM

**Review by d-rita — DISMISSED** [review]

**Comment by d-rita** [comment]
@kdaglas why is our coverage dropping?

File: authors/apps/articles/views/views.py [hunk 1]
```diff
@@ -175,7 +175,6 @@ def get(self, request, slug):
 class TagListAPIView(generics.ListAPIView):
     """ Create a view that is used to fetch all the tags """
     pagination_class = None
```
Comment (ja-odur) [inline authors/apps/articles/views/views.py:177]: @kdaglas please set the permission class to allowany instead of none

**Review by ja-odur — DISMISSED** [review]

File: authors/apps/articles/views/views.py (hunk 1 above)
Comment (kdaglas) [reply authors/apps/articles/views/views.py:177]: I have done that was pushing... thank you

**Review by mariamiah — DISMISSED** [review]

**Review by malfahad — DISMISSED** [review]

**Review by calvinpete — DISMISSED** [review]
LGTM

**Review by armstrongsouljah — DISMISSED** [review]
LTGM

**Review by MuhweziDeo — APPROVED** [review]

**Review by armstrongsouljah — APPROVED** [review]
LGTM

**Review by ja-odur — APPROVED** [review]
LGTM

**Pull request merged**